scratch/tpscompare/00-tpscompare.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
Also, just wondering if you tried zipping them up? Maaaaybe it would bring the size down (enough to bother)?

Reply:
> Also, just wondering if you tried zipping them up? Maaaaybe it would bring the size down (enough to bother)?

Git zips things up itself before store anything in the repo.

Reply:
> I noticed a decent set of DICOM files are now committed. Are you sure you want to include them? I did think we were trying to minimise this sort of thing in the repo... but maybe this matters less after the big split?

It does matter, primarily for the pain it introduces for new users cloning the repo as well as every CI needs to download the latest repo version so it has pushed the cloning stage on Azure to 1 minute.

... I think we do need working examples, and I think we do need real data... I think there might be value in reviving Zenodo for file storage and then deleting every file in the git history that is > 10 MiB, and then putting commit check that won't let files > 10 MiB be commited.

I have been thinking about a neat way to ease the use of Zenodo. Essentially making a data utility package that we can use to download demo data. Have the data utility cache the downloads and have a record of the download file hash.

Even with git VFS I am beginning to suspect that might still be a neater solution.

Reply:
> Git zips things up itself before storing anything in the repo.

Fair enough. Can we unzip some of the currently zipped data then? Might (slightly) speed up tests...

> If the tests and or docs still need those files, then they would still need to be downloaded. That's why it would be a good idea to implement a robust cache and tell Netlify as well as Azure where that cache is stored so they don't have to redownload those files every time...

I'm not so sure about the docs. Can the docs not point to files stored elsewhere? Then developers need not download those files to their local repos. I don't see why developers necessarily need fully working docs locally. If they're trying to understand something, they can refer to the online docs.

Reply:
> Fair enough. Can we unzip some of the currently zipped data then? Might (slightly) speed up tests...

This will result in those files being committed again, bloating the repo further, also GitHub has a hard limit on 100 MiB files, which most of the time I have zipped things up it has been to circumvent that limit...

Reply:
> I'm not so sure about the docs. Can the docs not point to files stored elsewhere? Then developers need not download those files to their local workstations unless they want to work on examples in particular. Otherwise, I don't see why developers necessarily need local, fully working docs by default. If they're trying to understand something, they can refer to the online docs.

I believe this was addressed in the Zenodo issue which will solve separate to this pull request.